yarn.lock

```diff
@@ -0,0 +1,9125 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Oh this project won't need yarn so we can delete this file. Yay for a lean codebase!